LICENSE

```diff
@@ -0,0 +1,40 @@
+SPRYKER SYSTEMS GMBH EVALUATION LICENSE AGREEMENT
```

Comment:
Replace the content of the file, use `vendor/spryker/decimal-object/LICENSE` as a source.